lib/js-yaml/loader.js

```diff
@@ -1100,10 +1102,11 @@ function readTagProperty(state) {
     return false;
   }
 
-  if (null !== state.tag) {
+  if (null !== state.tag && !atLineEnd) {
```

Comment:
If I understood this correctly, the you must place this block after `if (atLineEnd)` block instead of disabling it. In current state it would cause a bug allowing to place two or more tags in case of atLineEnd=true.